examples/chat/main.go

```diff
@@ -522,7 +522,7 @@ func addReplicatorCmd(id thread.ID, addrStr string) (out string, err error) {
 	if err != nil {
 		return
 	}
-	pid, err := net.AddReplicator(ctx, id, addr)
+	pid, err := net.AddReplicator(ctx, id, addr, peer.ID(""))
```

Comment:
Using an invalid peer id/log id like this just defaults to the current behavior.